eos-remove-old-flatpak-extension.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Remove old Flatpak unmanaged extensions directory
+DefaultDependencies=false
+Conflicts=shutdown.target
+Before=systemd-tmpfiles-setup.service
+After=local-fs.target
+Wants=local-fs.target
+
+# Only run on ARM systems where the old directory exists
+ConditionArchitecture=arm
+ConditionPathIsSymbolicLink=!/var/lib/flatpak/extension
+ConditionPathIsDirectory=/var/lib/flatpak/extension
+ConditionPathExists=/var/lib/flatpak/extension/org.freedesktop.Platform.GL.host
+
+[Service]
+Type=oneshot
+RemainAfterExit=true
+ExecStart=/usr/bin/rm -rf /var/lib/flatpak/extension
```

Comment:
I think we also need to clear up /var/endless-extra/flatpak/extension as well, to undo the effects of https://github.com/endlessm/eos-boot-helper/blob/a7ba47d258420502ba828e9493e01229e5af9e57/eos-link-host-mali-driver#L33-L34 on split systems.

Reply:
Yes, good catch! Fixed now.